app/lib/imports/journeys_missing_vehicle.rb

```diff
@@ -38,9 +38,9 @@ def results
   def records
     @records ||= CSV.table(csv_path).map do |record|
       {
-        journey_id: record[columns.fetch(:journey_id)],
-        move_id: record[columns.fetch(:move_id)],
-        vehicle_registration: record[columns.fetch(:vehicle_registration)],
+        journey_id: record[columns.fetch(:journey_id)].strip,
```

Comment:
Minor gripe - test coverage?

Reply:
Fair point, I've reworked this PR to introduce a class which handles the mapping of columns and encapsulates the `strip` and `downcase` logic so it could be tested on its own.